File: DSA/RATNA_KHANNA_590013141/day5question3.c
#include<stdio.h>
int main()
{
    int n;
    printf("enter number of elements in array\n");
    scanf("%d",&n);
    int nums[n];
    printf("enter the elements to the array\n");
    for(int i=0;i<n;i++)
    {
        scanf("%d",&nums[i]);
    }
    int j=n-1;
    int c=0;
    for(int i=0;i<n/2;i++)
    {
        if(nums[i]==nums[j])
        c++;
        j--;
    }
    int l=n/2;
    if(c==l)
    printf("true");
    else
    printf("false");
return 0;
}

File: DSA/Sarthak_Jain_590014936/DAY_5/Day_5_Question_3.py
#Sarthak Jain 590014936
#Day 5 Question 3
#Check if Array is Palindrome

nums = []

n = int(input("Enter number of elements:"))
for i in range(n):
    b = int(input("Enter element:"))
    nums.append(b)

print("Given List:", nums)

# Check if list is equal to its reverse
if nums == nums[::-1]:
    print("true")
else:
    print("false")
